interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
+        // Determine what text to show as a tooltip for this search_item.
+        if (SampleBin.hasItem(search_item)) {
+          return 'already added to analysis';
```

Comment:
Ah, so I just talked to @jaclyn-taroni, and it does sound like capitalizing the first letter would be a good idea. Additionally, we were discussing putting the word `ADD` in all caps as well.